### PR TITLE
feat: add toast component

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -772,3 +772,26 @@
 .rating{
     height: 1.5rem;
 }
+
+.toast{
+    display: flex;
+    background-color: var(--white);
+    padding: 0.5rem 1rem;
+    flex-direction: row;
```
Comment (JantuDeb) [inline components.css]: it is row by default

**Review by JantuDeb — APPROVED** [review]
this is Looks fine

File: components.css [hunk 2]
```diff
@@ -772,3 +772,26 @@
...
+}
+
+.toast-close-btn{
+    font-weight: bolder;
+    font-size: 1.5rem;
+}
+
+.toast-message{
+    font-weight: bolder;
+    flex-grow: 1;
```
Comment (sushil-mod) [inline components.css:795]: i like you used flex property

**Review by sushil-mod — APPROVED** [review]
toast looking fine in UI , good short and clean code

**Pull request merged**